_posts/2021-05-26-set-redirect-rules-in-nginx-based-wordpress-image.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "How to set the redirect rules in Nginx based Wordpress Image"
+author_name: "Arjun Baliga"
+tags:
+    - nginx
+    - Wordpress
+    - redirect rules
+categories:
+    - Azure App Service on Linux
+    - PHP
+    - WordPress
+    - Configuration
+header:
+    teaser: "/assets/images/WordPress.png" # There are multiple logos that can be used in "/assets/images" if you choose to add one.
+toc: true
+toc_sticky: true
+date: 2021-01-28 16:00:00
+---
+In the Nginx based WordPress image, if you'd like to configure the redirect rules please follow the steps below -
```

Comment:
"Our Wordpress on Linux image is Nginx based and if you'd like to configure the redirect rules please follow the steps below"